### ISSUE TITLE
Created a python package to extract data and generate reports 📈

### ISSUE THREAD

**Issue by AaronWard** [CSSEGISandData/2019-nCoV#15]
Had this working for the google sheets, but then decided to update for the github version

https://github.com/AaronWard/coronavirus-analysis

**What does it do?**
- Extracts latest entry for each date from 2018-nCoV
- creates an aggregated time series dataframe
- creates a summary report diagrams to visualize the growth in confirmed cases, deaths and recoveries

<br>

If you just want to see visualizations, i update this repo daily so star the repo and check the readme👍

**Comment by ciscorucinski** [comment]
Your "daily" numbers are not truly daily. You should grab the last values from the day to get more correct daily numbers

**Comment by AaronWard** [comment]
@ciscorucinski 
Can you point out where?

I'm only using the latest csv for each date, however 11th of feb is incorrect due to the reports not being complete yet.

**Comment by dawenx** [comment]
i agree with @ciscorucinski , for example, on https://github.com/CSSEGISandData/COVID-19/blob/master/daily_case_updates/02-08-2020_2304.csv, the value for Hubei was updated on 02/09/2020 at 3:03am and it's **27100, note that this ** **should NOT** be the value for Hubei on 02-08-2020 any more. If you look at https://github.com/CSSEGISandData/COVID-19/blob/master/daily_case_updates/02-09-2020_1030.csv, I understand why Hubei has 27100 on 2020-02-09 11:13:28.

The problem is: when people like myself uses the last file for a given day, i.e., 02-08-2020_2304, to set "final" number for that day, it's not correct any more. @AaronWard

**Comment by AaronWard** [comment]
@dawenx Thanks for the explanation, noted!

I will work on a fix tomorrow 👍

**Comment by AaronWard** [comment]
@dawenx @ciscorucinski 
Issue fixed 💯 
The code is now using the actually date values in the CSV's and keeping the latest entry per country / province

**Comment by brainwane** [comment]
Now [installable via PyPI](https://pypi.org/project/covidify/).
